package.json

```diff
@@ -1194,6 +1199,11 @@
                 "title": "Restart",
                 "category": "Docker Containers"
             },
+            {
+                "command": "vscode-docker.containers.select",
```

Comment:
So this command will be hard-coded in people's launch.json file, which means we should be very deliberate with the name because changing it in the future would break people.

I vote `docker.pickContainer` for two reasons:
1. I never understood why we had `vscode-docker` instead of just `docker` as the prefix. `vscode-` should be implied
1. `pickContainer` feels the most consistent with `pickRemoteProcess` and `pickProcess` from the C# extension

Reply:
1. I decided to take this extension's current prefix so keep consistency. Do you still want the name changed?
2. pickProcess and pickRemoteProcess doesn't actually exist. They use the variables to redirect to these commands: https://github.com/OmniSharp/omnisharp-vscode/blob/cfec123909a05d88ac625ec72aa0351fe74435ef/package.json#L891-L894. I do like pickContainer though.

Reply:
1. I'm not worried about consistency with the current command ids because those command ids aren't used directly by users (with a few minor exceptions).
1. Can we use a variable ourselves? Or would that have to be defined directly in the Omnisharp extension?

Reply:
> I never understood why we had vscode-docker instead of just docker as the prefix. vscode- should be implied

There can be other docker extensions, we should be careful about being too generic, we've conflicted with other extensions before.  vscode-docker ain't perfect, but it does somewhat imply an "official" docker extension.

Reply:
@EricJizbaMSFT as I understand it, omnisharp would have to declare the variable for it to be a default for the omnisharp "clr" or "coreclr" debugger types. I would think this couples this command with omnisharp. Don't think that would be good.

Reply:
> as I understand it, omnisharp would have to declare the variable for it to be a default for the omnisharp "clr" or "coreclr" debugger types. I would think this couples this command with omnisharp. Don't think that would be good.

Agreed we shouldn't couple this with Omnisharp 

> There can be other docker extensions, we should be careful about being too generic, we've conflicted with other extensions before. vscode-docker ain't perfect, but it does somewhat imply an "official" docker extension.

Honestly I would expect the generic one to be the official one and other arbitrary prefixes (aka `vscode-`) to be the non-official ones

Reply:
Consensus doesn't seem to be here, I'll wait for a decision. As it stands, I would be inclined towards: vscode-docker.containers.pick. I used .select because it was the same term as the Omnisharp one (consistency).

Reply:
For some reason, I though I had picked "select" because of Omnisharp... I was wrong, here are the command ids: 
- csharp.listProcess
- csharp.listRemoteProcess

I don't full agree with their ids since they return the selected process' id... which has nothing to do with listing in my mind.

Reply:
@EricJizbaMSFT I'd be fine with using `vscode-docker` here, consistent with the old way. It's essentially harmless to have the longer name including `vscode-`, and I'm a big fan of consistency when there isn't a very compelling reason to go against it.